src/shared/ui/Header/Header.tsx

```
@@ -3,36 +3,37 @@
-        <nav>파일 다운</nav>
-      </div>
+      </motion.button>
       <div className="flex items-center gap-2">
         {/* <Dropdown
           options={[
             { label: '한국어', value: '한국어' },
             { label: 'Eng', value: 'Eng' },
           ]}
         /> */}
-        <IconButton variant="secondary" size="lg" icon="IC_Account" ariaLabel="마이페이지" />
+        <LinkButton icon='IC_Image_Folder' label='마스코트 모음' href='/files' radius='full' variant='secondary' />
```

ESLint

[error] Replace `·icon='IC_Image_Folder'·label='마스코트·모음'·href='/files'·radius='full'·variant='secondary'` with `⏎··········icon="IC_Image_Folder"⏎··········label="마스코트·모음"⏎··········href="/files"⏎··········radius="full"⏎··········variant="secondary"⏎·······`

(prettier/prettier)